### DISCUSSION TITLE
Login token

### DISCUSSION THREAD

**Discussion by ivsaneesh** [vendurehq/vendure#1655] [Q&A]
Hi, 
Iam  sending the request for login using
`mutation{
  login(username:"sam@mail.com",password:"12345678",rememberMe:true){
    __typename
    ...on CurrentUser{
      identifier
      channels{
        id
        token
        code
        permissions
        __typename
      }
    }
  }
}`

and i am getting response

`{
  "data": {
    "login": {
      "__typename": "CurrentUser",
      "identifier": "sam@mail.com",
      "channels": [
        {
          "id": "1",
          "token": "lbm6ltowtmpodo0uj8o",
          "code": "__default_channel__",
          "permissions": [
            "Authenticated"
          ],
          "__typename": "CurrentUserChannel"
        }
      ]
    }
  }
}`

After login when i call 'me' api using the token <  "token": "lbm6ltowtmpodo0uj8o",>  it is not working

`{
  me {
    id
  }
}`

header i tried
`Authorization:Bearer lbm6ltowtmpodo0uj8o`
`Authorization:bearer lbm6ltowtmpodo0uj8o`

i am getting the message
`{
  "errors": [
    {
      "message": "You are not currently authorized to perform this action",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "me"
      ],
      "extensions": {
        "code": "FORBIDDEN"
      }
    }
  ],
  "data": {
    "me": null
  }
}`

i am using bearer token
` tokenMethod: 'bearer'`

is this the right token i am sending or not.

How exactly do i get the correct token.

**Comment by michaelbromley** [comment]
Hi,

Check these docs: https://www.vendure.io/docs/storefront/managing-sessions/#bearer-token-sessions

Specifically: 

> Certain mutations and queries initiate a session (e.g. logging in, adding an item to an order etc.). When this happens, the response will contain a HTTP header which [by default is called 'vendure-auth-token'](https://www.vendure.io/docs/typescript-api/auth/auth-options/#authtokenheaderkey).

So you don't want to use that "token" value from the response body - that is related to the Channel. Instead, inspect the response headers for the `vendure-auth-token` header - that contains the value you need.
